integration_tests/src/main/python/delta_lake_delete_test.py

```diff
@@ -139,12 +138,15 @@ def read_parquet_sql(data_path):
 
     assert_gpu_and_cpu_are_equal_collect(read_parquet_sql(data_path))
 
+fallback_readers_pre_353=["PERFILE", "MULTITHREADED", "COALESCING"]
+fallback_readers_353_plus=["COALESCING"]
```

Comment:
Why do we fall back for the `COALESCING` reader here? Based on [this](https://github.com/NVIDIA/spark-rapids/pull/13491/files#diff-76c3258687500831840e30d43a5adf84f0f51ac2a7606c48a8496c5496afe79cR221-R224), should it still run on GPU but with multithreaded instead?

Reply:
According to [your previous comment](https://github.com/NVIDIA/spark-rapids/pull/13491#issuecomment-3335116928), this test seems to have passed with `COALESCING` fallback in your previous testing. Why did it pass? Can you make sure that this test passes after your last update?